build.gradle

```diff
@@ -24,6 +22,6 @@ task clean(type: Delete) {
 }
 
 task wrapper(type: Wrapper) {
-	gradleVersion = '3.3'
+	gradleVersion = '4.0'
```

Comment:
@trevjonez Could this be causing the problem Steve is referencing? I'm also getting the same issue.  I wonder if its because I'm using Android Studio 2.3.3 and this is using gradle and build tools for Android Studio 3.0